### PR TITLE
[EM-1767] Distinguish local & national teams

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

**Review by tony-tran — APPROVED** [review]

File: features/api/teams.feature [hunk 1]
```diff
@@ -19,12 +19,86 @@ Feature:
...
+
+
+
+
+
+
+
+
+
+  Scenario: As a user granted with national scope, I can get the list of national teams only
```
Comment (ottaviano) [inline features/api/teams.feature:95]:
```suggestion
    Then the response status code should be 404

  Scenario: As a user granted with national scope, I can get the list of national teams only
```

File: src/Admin/Extension/EntityScopeVisibilityAdminExtension.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            ])
+        ;
+    }
+
+    public function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper
+            ->add('visibility', null, [
+                'label' => 'Visibilité',
+                'template' => 'admin/scope/_list_visibility.html.twig',
```
Comment (ottaviano) [inline src/Admin/Extension/EntityScopeVisibilityAdminExtension.php:51]:
```suggestion
                'template' => 'admin/list/list_visibility.html.twig',
```

File: src/Admin/Extension/EntityScopeVisibilityAdminExtension.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            ])
+            ->add('zone', null, [
+                'label' => 'Zone',
+                'template' => 'admin/scope/_list_zone.html.twig',
```
Comment (ottaviano) [inline src/Admin/Extension/EntityScopeVisibilityAdminExtension.php:55]:
```suggestion
                'template' => 'admin/list/list_zone.html.twig',
```

**Review by ottaviano — APPROVED** [review]

File: src/Validator/Scope/ScopeVisibilityValidator.php [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                ->buildViolation($constraint->localScopeWithUnmanagedZoneMessage)
+                ->atPath('zone')
+                ->addViolation()
+            ;
+        }
+    }
+
+    private function getAdherent(): Adherent
+    {
+        return $this->security->getUser();
```
Comment (elvino29) [inline src/Validator/Scope/ScopeVisibilityValidator.php:92]: check if current user instance of Adherent

File: src/Entity/Team/Team.php [hunk 5]
```diff
@@ -113,7 +123,7 @@ class Team implements EntityAdherentBlameableInterface, EntityAdministratorBlame
      */
     private $members;
```
Comment (tony-tran) [inline src/Entity/Team/Team.php:124]: You can add the type of members

**Review by elvino29 — APPROVED** [review]

File: src/Entity/Team/Team.php (hunk 5 above)
Comment (Remg) [reply src/Entity/Team/Team.php:124]: thank you @tony-tran

**Pull request merged**